frontend/src/components/bs/components/spaces-table/spaces-table.tsx

```diff
@@ -1,20 +1,52 @@
 import { FC } from 'react';
-import { useAppSelector, useMemo } from 'hooks/hooks';
+import {
+  useAppSelector,
+  useMemo,
+  useAppDispatch,
+  usePagination,
+} from 'hooks/hooks';
 import { Table } from 'components/common/common';
 import { getRows, getColumns } from './helpers/helpers';
 import { DataStatus } from 'common/enums/enums';
+import { Pagination } from 'common/enums/enums';
```

Comment:
```suggestion
import { DataStatus, Pagination  } from 'common/enums/enums';
```
in every service table component

Reply:
agreed